pkg/component/delete/delete.go

```diff
@@ -74,3 +85,85 @@ func references(list []unstructured.Unstructured, ownerRef metav1.OwnerReference
 	}
 	return false
 }
+func (do DeleteComponentClient) ListResourcesToDeleteFromDevfile(devfileObj parser.DevfileObj, appName string) (map[string]string, []unstructured.Unstructured, error) {
+	var resources []unstructured.Unstructured
+	var isInnerLoopLabels map[string]string
+	componentName := devfileObj.GetMetadataName()
+
+	deployment, err := do.kubeClient.GetDeploymentByName(fmt.Sprintf("%s-%s", componentName, appName))
```

Comment:
It would be important to use a centralized function wrapping the `fmt.Sprintf("%s-%s", componentName, appName)`, that is also used during deployment creation, so we can change the name of the deployment easily.

As far as I can see, it is `pkg.util.NamespaceKubernetesObject` that is used during deployment creation. Please use this function, and feel free to change its name to make it more expressive. You could also place it in the `pkg/component` package

Reply:
I saw `pkg.util.NamespaceKubernetesObject` yday, but it seemed to be doing more than just appending the two names, so I decided to not use it, but sure I can use it.